src/main/java/com/powsybl/openloadflow/network/impl/LfBusImpl.java

```diff
@@ -156,14 +160,18 @@ private double checkTargetV(double targetV) {
 
     void addLoad(Load load) {
         loads.add(load);
-        this.loadTargetP += load.getP0();
-        this.loadTargetQ += load.getQ0();
+        initialLoadTargetP = load.getP0();
```

Comment:
I think it should be an addition here: maybe we should have a unit test with several loads on one bus.

Reply:
Yes, you are right. I am also going to add a unit test.